src/geometry.jl

```diff
@@ -1,59 +1,93 @@
 """
     Shape
 
-Abstract type for the shape of the object being modelled.
+Abstract supertype for the shape of the organism being modelled.
 """
 abstract type Shape end
 
+"""
+    Insulation
+
+Abstract supertype for the shape of the organism being modelled.
+"""
 abstract type Insulation end
 
-abstract type AbstractBody end
+"""
+    Naked <: Insulation
 
+    Naked()
+
+Insulation trait for an organism without fur.
+"""
 struct Naked <: Insulation end
 
+"""
+    Fur <: Insulation
+    
+    Fur(thickness)
+
+Insulation trait for an organism with fur.
+"""
 struct Fur{T} <: Insulation
     thickness::T
 end
 
+"""
+    Geometry
+
+    Geometry(volume, characteristic_dimension, lengths, area)
+
+The geometry of an organism.
+"""
 struct Geometry{V,C,L,A}
     volume::V
     characteristic_dimension::C
     lengths::L
     area::A
 end
 
+"""
+    AbstractBody
+
+Abstract supertype for organism bodies.
+"""
+abstract type AbstractBody end
+
+shape(body::AbstractBody) = body.shape
+insulation(body::AbstractBody) = body.insulation
+geometry(body::AbstractBody) = body.geometry
+calc_area(body::AbstractBody) = calc_area(shape(body), body)
+
+"""
+    calc_silhouette_area(body::AbstractBody, θ)
+
```

Comment:
change to: "Calculates the silhouette (projected) area of a cylinder given a solar zenith angle, θ."

Reply:
```suggestion

Calculates the silhouette (projected) area of a cylinder given a solar zenith angle, θ.
```